src/components/PostInfo/PostInfo.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { UserInfo } from '../UserInfo';
+
+import './PostInfo.scss';
+import { CommentInfo } from '../CommentInfo';
+import { Post } from '../../types/Post';
+
+type Props = Post;
+
+export const PostInfo: React.FC<Props> = ({
+  id, title, body, user, comments,
+}) => {
+  return (
+    <article className="post App__posts-item">
+      <div className="post__id-wrapper">
+        <span className="post__id">{id}</span>
+      </div>
+
+      <h2 className="post__title">{title}</h2>
+
+      <p className="post__body">{body}</p>
+
+      {user && <UserInfo name={user.name} email={user.email} />}
+
+      {comments && <CommentInfo comments={comments} />}
```

Comment:
Now remove this unnecessary check or make it logical with checking array's length

Reply:
Why unnesessary? I don't undestand, it doesn't work without this check, because method find can be undefined.

Reply:
![image](https://user-images.githubusercontent.com/74960600/145051064-cf1c0a33-09f1-41c9-a32c-c12bfc103ab7.png)

Reply:
I commented on your `comments` check